OracleDatabase/SingleInstance/extensions/prebuiltdb/README.md

```diff
@@ -41,4 +42,8 @@ This extended image would be very useful in CI/CD scenarios, where database woul
 
 Some limitations are listed as follows:
 - **External volume can not be used** for database persistence (as data files are inside the image itself).
-- In Kubernetes environment, **the single replica mode** (i.e. replicas=1) can be used for database deployments.
\ No newline at end of file
+- In Kubernetes environment, **the single replica mode** (i.e. replicas=1) can be used for database deployments.
+- The database created will not use more than 2GB of memory.
+  - The amount of memory allocated for the database is calculated during creation of the database, which runs during the docker build phase. By default, `AUTO_MEM_CALCULATION` is `false` so that the container built with this extension is portable, but it also means that the database inside of the containers deployed with this image will not use more than 2GB, even if more is available/allocated.
+  - If you know the environment where you plan to deploy the container built with this extension, you can build the image with `DOCKER_BUILDKIT=0 docker build --memory=4096m --build-arg AUTO_MEM_CALCULATION=true` so that the database is created with 4GB of memory allocated.
```

Comment:
Setting memory at build time seems to be ineffective. The prebuiltdb now always have 2GB memory for the database. Steps executed:
`./buildExtensions.sh -x prebuiltdb -b oracle/database:21.3.0-ee -o '--memory 6g --bui│ORCLCDB.__unified_pga_pool_size=0
ld-arg AUTO_MEM_CALCULATION=true'`
The above command used for building the prebuiltdb image.
Below is the content of spfile of the image, for getting total memory:
```bash
docker run -dt --name t1 oracle/database:ext
23a7800405a1f7c77cd9023c4f9aff1f8694bd4e4dfa19d80135a32683b869fa
[opc@prebuiltdb-test dockerfiles]$ docker exec -it t1 /bin/bash
bash-4.2$ cd /opt/oracle/oradata/dbconfig/ORCLCDB/
bash-4.2$ ls
listener.ora  orapwORCLCDB  oratab  spfileORCLCDB.ora  sqlnet.ora  tnsnames.ora
bash-4.2$ cat spfileORCLCDB.ora
C"ʱACC"p\ORCLCDB.__data_transfer_cache_size=0
ORCLCDB.__db_cache_size=973078528
ORCLCDB.__inmemory_ext_roarea=0
ORCLCDB.__inmemory_ext_rwarea=0
ORCLCDB.__java_pool_size=0
ORCLCDB.__large_pool_size=100663296
ORCLCDB.__oracle_base='/opt/oracle'#ORACLE_BASE set from environment
ORCLCDB.__pga_aggregate_target=536870912
ORCLCDB.__sga_target=1610612736
ORCLCDB.__shared_io_pool_size=0
ORCLCDB.__shared_pool_size=520093696
ORCLCDB.__streams_pool_size=0
ORCLCDB.__unified_pga_pool_size=0
*.audit_file_deCC"      st='/opt/oracle/admin/ORCLCDB/adump'
*.audit_sys_operations=false
*.audit_trail='none'
*.compatible='21.0.0'
*.control_files='/opt/oracle/oradata/ORCLCDB/control01.ctl'
*.db_block_size=8192
*.db_name='ORCLCDB'
*.diagnostic_dest='/opt/oracle'
*.dispatchers='(PROTOCOL=TCP) (SERVICE=ORCLCDBXDB)'
*.enable_pluggable_database=true
*.local_listener=''
*.nls_language='AMERICAN'
*.nls_territory='AMERICA'
*.open_cursors=300
*.pga_aggregate_target=512m
*.processes=1600
*.remote_login_passwordfiCC"    =le='EXCLUSIVE'
*.sga_target=1536m
*.undo_tablespace='UNDOTBS1'
bash-4.2$ cd /opt/oracle/oradata/dbconfig/ORCLCDB/
```
As one can see that sga_target 1536m and pga_target is 512m; total 2GB.

Reply:
Can you pls confirm this, we might need to remove this line from the README. Rest all looks fine.

Reply:
Wasn't `total_memory` meant to appear in your example?

Reply:
If `--build-arg AUTO_MEM_CALCULATION=true` for prebuiltdb is not being effective, then we have a bug.

Reply:
Anyway, I will try to replicate here to check.

Reply:
@abhisbyk I identified what happened in your example. Most likely you forgot `DOCKER_BUILDKIT=0` as I mentioned in the doc.

Watch about about builx as well. If you have buildx, you have to first run `docker buildx uninstall`.

See https://github.com/docker/buildx/issues/957

Reply:
Here:

```
❯ DOCKER_BUILDKIT=0 ./buildExtensions.sh -x prebuiltdb -b oracle:21.3.0 -t oracle:21.3.0-prebuiltdb -o '--memory 3g --build-arg AUTO_MEM_CALCULATION=true'
❯ docker run --rm -it oracle:21.3.0-prebuiltdb cat /opt/oracle/oradata/dbconfig/ORCLCDB/spfileORCLCDB.ora
C"4��ACC",ORCLCDB.__data_transfer_cache_size=0
ORCLCDB.__db_cache_size=1644167168
ORCLCDB.__inmemory_ext_roarea=0
ORCLCDB.__inmemory_ext_rwarea=0
ORCLCDB.__java_pool_size=0
ORCLCDB.__large_pool_size=83886080
ORCLCDB.__oracle_base='/opt/oracle'#ORACLE_BASE set from environment
ORCLCDB.__pga_aggregate_target=805306368
ORCLCDB.__sga_target=2415919104
ORCLCDB.__shared_io_pool_size=117440512
ORCLCDB.__shared_pool_size=553648128
ORCLCDB.__streams_pool_size=0
ORCLCDB.__unified_pga_pool_size=0
*.auditCC"$_file_dest='/opt/oracle/admin/ORCLCDB/adump'
*.audit_sys_operations=false
*.audit_trail='none'
*.compatible='21.0.0'
*.control_files='/opt/oracle/oradata/ORCLCDB/control01.ctl'
*.db_block_size=8192
*.db_name='ORCLCDB'
*.diagnostic_dest='/opt/oracle'
*.dispatchers='(PROTOCOL=TCP) (SERVICE=ORCLCDBXDB)'
*.enable_pluggable_database=true
*.local_listener=''
*.nls_language='AMERICAN'
*.nls_territory='AMERICA'
*.open_cursors=300
*.pga_aggregate_target=768m
*.processes=1280
*.remote_login_paCC"(sswordfile='EXCLUSIVE'
*.sga_target=2304m
*.undo_tablespace='UNDOTBS1'
```

You can see that it inherited the 3GB instead of using the default of 2GB.

@abhisbyk I think the PR is good then.

Reply:
Yes, it was because of buildx. Things working fine for me now.